<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceDetectionService_video</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnClustering">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clustering</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGallery">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gallery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProbe">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Probe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editGallery">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editProbe">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEval">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eval</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="editEval">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>411</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnStartService">
         <property name="text">
          <string>Start Service</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>405</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChangeDBPath">
   <property name="text">
    <string>Change DB Path</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
